### ISSUE TITLE
How to choose the best trained model

### ISSUE THREAD

**Issue by menghuaa** [MCG-NJU/MMN#2]
Hi, how do I select a model for testing when I have trained it? Does it rely on the loss or the test effect on the validation set? Your code does not give how to choose the best model.

**Comment by zhenzhiwang** [comment]
Thanks for your interest of our work. For Charades-STA which has no validation set, you can directly select the best epoch in the test set. For TACoS and ActivityNet, you can select the epoch with the best performance (e.g., a weighted sum of these metrics) on validation set. In our practice, we use a sum of these metrics on validation set in the tensorboard to manually select epoch. For simplicity of our code, we delete the tensorboard part of the released code. Please implement it by yourself.
